eosmetrics/emtr-event-recorder.c

```diff
@@ -304,6 +311,19 @@ send_events_to_dbus_finish_callback (EmerEventRecorderServer *dbus_proxy,
     }
 }
 
+/* Check the EOS_DISABLE_METRICS environment variable to see if we should
+ * skip submitting any metrics. This is intended to be set when running unit
+ * tests in other modules, for example, to avoid submitting metrics from unit
+ * test runs. */
+static gboolean
+disable_event_submission (void)
+{
+  const gchar *val = g_getenv ("EOS_DISABLE_METRICS");
+
+  return (val != NULL &&
+          (g_strcmp0 (val, "") == 0 || g_strcmp0 (val, "1") == 0));
```

Comment:
If you've already done the NULL check then you can probably use regular `strcmp` here.